Jenkinsfile

```diff
@@ -221,21 +222,38 @@ pipeline {
         }
         stage('Check PR') {
             when { changeRequest() }
-            steps {
-                catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS',
-                           message: "PR did not get committed with required git hooks.  Please see utils/githooks/README.md.") {
-                    sh 'if ! ' + cachedCommitPragma('Required-githooks', 'false') + '''; then
-                           echo "PR did not get committed with required git hooks.  Please see utils/githooks/README.md."
-                           exit 1
-                        fi'''
-                }
-            }
-            post {
-                unsuccessful {
-                    echo "PR did not get committed with required git hooks.  Please see utils/githooks/README.md."
+            parallel {
+                stage('Used Required Git Hooks') {
+                    steps {
+                        catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS',
+                                   message: "PR did not get committed with required git hooks.  Please see utils/githooks/README.md.") {
+                            sh 'if ! ' + cachedCommitPragma('Required-githooks', 'false') + '''; then
+                                   echo "PR did not get committed with required git hooks.  Please see utils/githooks/README.md."
+                                   exit 1
+                                fi'''
+                        }
+                    }
+                    post {
+                        unsuccessful {
+                            echo "PR did not get committed with required git hooks.  Please see utils/githooks/README.md."
+                        }
+                    }
+                } // stage('Used Required Git Hooks')
+                stage('Branch name check') {
+                    when { changeRequest() }
+                    steps {
+                        script {
+                            if (!env.CHANGE_BRANCH.contains('/')) {
+                                error('This PR was not created on a branch of the format $your_unique_prefix/' +
```

Comment:
I'd suggest linking to the relevant section of the developer document: https://daosio.atlassian.net/wiki/spaces/CI/pages/1359582959/The+New+CI+Pipeline#TheNewCIPipeline-TheGitHubWorkflow instead. e.g. `Your PR branch name does not follow the rules. Please rename it according to the rules described here: https://daosio.atlassian.net/l/cp/JV3Fsk0P`. This would allow us to adjust the text describing the rules without needing to update this script.

Reply:
The https://daosio.atlassian.net/l/cp/JV3Fsk0P anchor doesn't seem to be working unfortunately.  I was just taken to the top of the page clicking that link.

Reply:
Updated to use the tiny URL.